### DISCUSSION TITLE
Channel list disappeared

### DISCUSSION THREAD

**Discussion by monkeyman86** [iptv-org/community#548] [Q&A] [answered]
Hi, I set up a channel list in my Kodi app on my TV but today all channels disappeared suddenly. I checked the settings and the address and it all looks correct. do you know if there is an ongoing issue on accessing channels or lists?

**Comment by PopeyeTheSai10r** [comment]
Which playlist are you using? Are you having issues accessing it as of now?

**Reply by monkeyman86** [reply]
i am using xxx.m3u i tried few others from the same list on Kodi. however none of the lists load any channel.

**Reply by PopeyeTheSai10r** [reply, marked as answer]
The XXX playlist has been removed from the repo as per https://github.com/iptv-org/iptv/pull/15725.

All other .m3u files should work but make sure to check https://github.com/iptv-org/iptv?tab=readme-ov-file#playlists to know you have the correct url.
